lua/lspconfig/configs/pylyzer.lua

```diff
@@ -23,12 +23,25 @@ return {
         checkOnType = false,
       },
     },
+    ergPath = vim.fn.getenv('HOME') .. '/.erg',
+    on_new_config = function(new_config, _)
+      if new_config.ergPath then
+        new_config.cmd_env.ERG_PATH = new_config.ergPath
+      end
+    end,
```

Comment:
This saves the user basically nothing, while also adding a special-case interface to this config.

It would be more useful to just document what you're doing here, instead of adding a special `ergPath` field.

Reply:
I thought the `cmd_env` is gonna work? I found it in [bqnlsp's default config](https://github.com/neovim/nvim-lspconfig/blob/f012c1b176f0e3c71f40eb309bdec0316689462e/lua/lspconfig/configs/bqnlsp.lua#L24).

I think it would still be nice if I set the `ERG_PATH` in default config, so users who aren't familiar with lsp config can use it out of box:
```lua
    on_new_config = function(new_config, _)
      vim.fn.setenv("ERG_PATH", vim.fn.getenv("HOME") .. "/.erg")
    end
```
Is this OK?

Reply:
`cmd_env` is a standard field, no objection to using that. My objection is about the custom `ergPath` thing.



> still be nice if I set the `ERG_PATH` in default config, so users who aren't familiar with lsp config can use it out of box

Sure, yes. And some sort of brief hint in the docs.